kademlia/src/main/java/ingsw/group1/kademlia/NodesComparator.java

```diff
@@ -0,0 +1,28 @@
+package ingsw.group1.kademlia;
+
+import java.util.Comparator;
+
+/**
+ * Compares Nodes based on the distance to a target node
+ */
+public class NodesComparator implements Comparator<Node<BinarySet>> {
+    Node<BinarySet> targetNode;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/RicDeZen/ingsw-group1-kademlia/pullRequest?prid=4771188)